### DISCUSSION TITLE
Long DB Sync Times

### DISCUSSION THREAD

**Discussion by HPMSGEN8** [Chia-Network/chia-blockchain#14340] [Support]
Good Day

So i have moved to Docker with Machinaris and i copied over my v2 db that was synced only half way. It is now on July 2022.

Is there an Official location to download the latest DB. Also my v2 DB is over 80GB i think something is wrong or how do i compress this etc...

My wallet is synced. 

Thanks

**Comment by shaneo257** [comment]
my fully synced v2 db is almost 100gb, and no, theres isn't a official location to download the latest db

**Reply by HPMSGEN8** [reply]
Thanks alot
